cmd/helm/install.go

```diff
@@ -142,6 +142,7 @@ func addInstallFlags(f *pflag.FlagSet, client *action.Install, valueOpts *values
 	f.BoolVar(&client.Atomic, "atomic", false, "if set, installation process purges chart on fail. The --wait flag will be set automatically if --atomic is used")
 	f.BoolVar(&client.SkipCRDs, "skip-crds", false, "if set, no CRDs will be installed. By default, CRDs are installed if not already present")
 	f.BoolVar(&client.SubNotes, "render-subchart-notes", false, "if set, render subchart notes along with the parent")
+	f.StringVar(&client.NameFlag, "name", "", "The release name (and omit NAME parameter)")
```

Comment:
I believe v2 also had a short flag `-n` set. so this should probably be:

```go
	f.StringVarP(&client.NameFlag, "name", "n", "", "The release name (and omit NAME parameter)")
```

```log
panic: unable to redefine 'n' shorthand in "install" flagset: it's already used for "name" flag [recovered]
	panic: unable to redefine 'n' shorthand in "install" flagset: it's already used for "name" flag
....
....
	/Users/bdd/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:895 +0x60c
helm.sh/helm/v3/cmd/helm.executeActionCommandC(0xc0001070a0, 0x2652554, 0x3c, 0x1062fa0, 0xc0004786f8, 0x56df0838, 0xc000461000, 0xc000478708)
	/Users/bdd/helm/cmd/helm/helm_test.go:116 +0x1f8
helm.sh/helm/v3/cmd/helm.runTestActionCmd.func1(0xc000738300)
	/Users/bdd/helm/cmd/helm/helm_test.go:83 +0x147
testing.tRunner(0xc000738300, 0xc0002e4400)
```
Unfortunately, this results in a panic in the test setup.